traefik/faency
# fix: kill css incompatible error

# Description

Closes #293 

Applies solutions from related issue on stitches.

# How to test

`Textarea.stories.tsx` had typing issues. I'm taking this one as example, but this PR should fix all `css` typing issues in the project. It would be awesome to validate that point with more use cases.

EDIT: I just added stories for `Text` and `Heading` showing that we can add `css` prop to those components :tada: 

1. Try opening the file without changes applied by my PR. Check you have typescript lint issues in the file
2. Checkout my PR, make sure you restart you IDE in-between (close/reopen)
3. Check you no longer have typescript lint issues in the file

# Preview

https://user-images.githubusercontent.com/3367393/154343211-da51ec58-e84d-490c-b567-37b3ada16249.mp4
